templates/robotics/ros-esm.html

```diff
@@ -44,203 +38,177 @@ <h4>
-      <div>
-
-        <div class="p-card u-align--left">
-          <h3 class="p-muted-heading">
-            Delivered in partnership with
-          </h3>
-          <hr class="u-sv1">
-          <div>
-            <a href="https://www.openrobotics.org/">
-              {{
-                image(
-                url="https://assets.ubuntu.com/v1/8da00274-open-robotics.png",
-                alt="",
-                height="61",
-                width="214",
-                hi_def=True,
-                attrs={ "style": "margin: 1rem 0;"},
-                loading="lazy",
-                ) | safe
-              }}
-            </a>
-          </div>
-          <p class="p-card__content">
-            Maintainer of the Robot Operating System and Gazebo
-          </p>
+  </div>
+  <hr class="is-fixed-width">
+  <div class="p-strip">
+    <div class="u-fixed-width">
+      <div class="row">
+        <p>ROS ESM security coverage</p>
+      </div>
+      <div class="row p-tiered-cards">
+        <div class="col-3 col-medium-3 p-tiered-card--low">
+          <h3>Main</h3>
+          <p class="p-card__content">2,300 packages in the Ubuntu Main repo supported for 5 years after EOL.</p>
+        </div>
+        <div class="col-3 col-medium-3 p-tiered-card--medium">
+          <h3>Universe</h3>
+          <p class="p-card__content">An additional 23,000+ packages in the Ubuntu Universe repository for 10 years.</p>
+        </div>
+        <div class="col-6 col-medium-6 p-tiered-card--high">
+          <h3>ROS ESM</h3>
+          <p class="p-card__content">More than 600 packages for ROS 1 Kinetic and Melodic and ROS 2 Foxy. Other ROS distributions will be added.</p>
         </div>
       </div>
     </div>
   </div>
+  <hr class="is-fixed-width">
 </section>
-
-<section class="p-strip--light">
-  <div class="u-fixed-width">
-    <div class="u-sv3 u-hide--small">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/894ffe5b-ESM+for+ROS+vertical.svg",
-        alt="",
-        width="748",
-        height="690",
-        hi_def=True,
-        loading="lazy",
-        ) | safe
-      }}
-    </div>
-    <h2>Security maintenance you can trust</h2>
-    <p>No more waiting for a patch to be made available, a breach identified or a security expert available to patch your software stack. With ROS ESM, the time consuming and resource intensive work of keeping core ROS packages secure is no longer a problem.</p>
-    <p>Our security experts follow a standardised process to identify vulnerabilities, create and test security patches. The implemented testing infrastructure reduces the possibility of reintroducing the vulnerability, and also improves the compatibility with our ESM repository.</p>
-    <p>Get instant updates and trouble-free maintenance for you and your robotic users with ROS ESM.</p>
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="row">
+    <p>A few of the 2,300 packages included in Ubuntu Main:</p>
```

Comment:
Any reason not to use the logo section component here?

Reply:
ah the size probably. Can we wrap in a p-strip.is-shallow.u-no-padding--top to insert some spacing betwene the lines and the logos?  We should have a version of the logo section for small logos, issue filed [here](https://github.com/canonical/vanilla-framework/issues/4642)

Reply:
Do you mean update the section classes or add a new strip inside?